.travis.yml

```diff
@@ -12,11 +12,15 @@ compiler:
 notifications:
     email:
         recipients:
-            - su2code-dev@lists.stanford.edu
+            - Thomas.Economon@us.bosch.com
```

Comment:
Please revert these for the merge into develop.

Reply:
Done